Civi/Sepa/Lock/SepaAsyncBatchLock.php

```diff
@@ -29,29 +29,30 @@
  */
 final class SepaAsyncBatchLock {
 
-  private string $name;
+  private ?string $acquiredId = NULL;
 
-  public function __construct(string $name) {
-    $this->name = $name;
+  private string $lockFile;
```

Comment:
`$lockFilePath` because it's not a file handle?

Reply:
Changed.